tests/unit/countones.ys

```diff
@@ -0,0 +1,96 @@
+logger -expect log "SAT proof finished - no model found: SUCCESS" 1
```

Comment:
Please make a `countones.sv` test out of this. All .sv files placed in tests/unit/ undergo proving of their asserts, see the existing tests

Reply:
I think you renamed the file. What I'm after is having a `tests/unit/countones.sv` test in the style of other .sv tests from the same directory